src/frontend/src/store/modules/answers.js

```diff
@@ -42,9 +44,8 @@ export default {
     },
     async UPDATE_ANSWER({ commit }, payload) {
```

Comment:
아래와 같이 받으면 payload를 중복해서 쓰지 않을 수 있을 것 같아요

```suggestion
    async UPDATE_ANSWER({ commit }, {answerId, updateContent}) {
```